docs/source/custom-collections.rst

```diff
@@ -560,9 +560,12 @@ implementation. There are two ways to do this:
 
 2. Register a function with ``dask.base.normalize_token``
 
-   If defining a method on the class isn't possible, you can register a tokenize
-   function with the ``normalize_token`` dispatch.  The function should have the
-   same signature as described above.
+   If defining a method on the class isn't possible or you need to
+   customize the tokenize function for a class who's super-class is
```

Comment:
```suggestion
   customize the tokenize function for a class whose super-class is
```

Reply:
I swear English is my first (and really only) language...